plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/panels/CodeModernizerBanner.kt

```diff
@@ -114,28 +115,23 @@ class CodeModernizerBanner(val project: Project) : JPanel(BorderLayout()) {
     }
 
     fun updateRunningTime(runTime: Duration?) {
-        try {
-            if (runTime == null) {
-                infoLabelRunningTime.text = ""
-            } else {
-                val timeTaken = runTime.toKotlinDuration().inWholeSeconds.seconds.toString()
-                infoLabelRunningTime.text = message(
-                    "codemodernizer.toolwindow.transformation.progress.running_time",
-                    timeTaken
-                )
-            }
-        } catch (exception: AlreadyDisposedException) {
-            LOG.warn { "Disposed when about to create the loading panel" }
-            return
+        infoLabelRunningTime.text = if (runTime != null) {
```

Comment:
Again here just curious, what running time is it referring to?

Reply:
The "42s" in the image above